packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -146,6 +146,26 @@ cluster.addCapacity('AsgSpot', {
 });
 ```
 
+### Bottlerocket
```

Comment:
I wonder if we should place the doc before the spot instances intro [here](https://github.com/aws/aws-cdk/pull/10097/files#diff-2d7fe3800f757a9d4c3f8c7c8e51d49bR127), since this is the place where we introduce how to add customized capacity. We could still make it a subsection to make this stand out. Let me know what you think about it.

Reply:
Agree. Let me update the README.